### ISSUE TITLE
The plugin is not compatible with Parcel 1.7.1

### ISSUE THREAD

**Issue by item4** [fathyb/parcel-plugin-typescript#36]
## Deps

```
parcel-bundler@1.7.1
typescript@2.8.3
```

## Error

```
/Users/item4/Projects/item4.github.io/node_modules/parcel-bundler/src/workerfarm/Worker.js:114
        throw new Error(
        ^

Error: Worker Farm: Received message for unknown index for existing child. This should not happen!
    at Worker.receive (/Users/item4/Projects/item4.github.io/node_modules/parcel-bundler/src/workerfarm/Worker.js:114:15)
    at ChildProcess.emit (events.js:180:13)
    at emit (internal/child_process.js:783:12)
    at process._tickCallback (internal/process/next_tick.js:114:19)
error Command failed with exit code 1.
```

## Temporary fix solution

remove parcel-plugin-typescript

## reproduce codebase

https://github.com/item4/item4.github.io

(sorry, I can not make simple case)

**Comment by jaredpetker** [comment]
This manifests itself differently with 1.8.1 with this error

```
  Cannot read property 'run' of undefined
    at /Users/jaredpetker/dev/cc/confidentwholesale/frontend/node_modules/parcel-bundler/lib/workerfarm/child.js:70:45
    at Generator.next (<anonymous>)
    at step (/Users/jaredpetker/dev/cc/confidentwholesale/frontend/node_modules/parcel-bundler/lib/workerfarm/child.js:3:191)
    at /Users/jaredpetker/dev/cc/confidentwholesale/frontend/node_modules/parcel-bundler/lib/workerfarm/child.js:3:437
    at new Promise (<anonymous>)
    at /Users/jaredpetker/dev/cc/confidentwholesale/frontend/node_modules/parcel-bundler/lib/workerfarm/child.js:3:99
    at Child.handleRequest (/Users/jaredpetker/dev/cc/confidentwholesale/frontend/node_modules/parcel-bundler/lib/workerfarm/child.js:77:7)
    at Child.messageListener (/Users/jaredpetker/dev/cc/confidentwholesale/frontend/node_modules/parcel-bundler/lib/workerfarm/child.js:38:19)
    at emitTwo (events.js:130:20)
    at process.emit (events.js:213:7)
```

**Comment by fathyb** [comment]
The bug was due to a weird side-effect in the new Parcel WorkerFarm, the error message misled me into thinking it was due to the plugin's IPC. Other plugins may be affected.

I released a (way too) simple fix for Parcel `1.8` in [`parcel-plugin-typescript@0.7.1`](https://github.com/fathyb/parcel-plugin-typescript/blob/develop/CHANGELOG.md#071-2018-05-05). I'm not 100% confident though, let me know if you encounter any similar issue!
